doc/rst/platforms/omnipath.rst

```diff
@@ -0,0 +1,159 @@
+.. _readme-omnipath:
+
+===========================
+Using Chapel with Omni-Path 
+===========================
+
+This document describes how to run Chapel across multiple nodes of
+an Intel Omni-Path cluster. :ref:`readme-multilocale` describes
+general information about running Chapel in a multilocale
+configuration.
+
+.. warning::
+    GASNet's ofi (libfabric) conduit is used to target Omni-Path.
+    This conduit is technically experimental, but has been stable in
+    our own testing.
+
+    Note that ``CHPL_COMM=ofi`` is not supported for Omni-Path
+
+.. contents::
+
+-------------------------
+Configuring for Omni-Path
+-------------------------
+
+To build Chapel with Omni-Path support, set:
+
+.. code-block:: bash
+
+      export CHPL_COMM=gasnet
+      export CHPL_COMM_SUBSTRATE=ofi
+
+GASNet should automatically detect the ``psm2`` provider, but to
+ensure this provider is used ``FI_PROVIDER=psm2`` may optionally be
+set.
+
+----------------------
+Configuring a Launcher
+----------------------
+
+A ``gasnetrun_ofi`` based launcher should be used to launch jobs and
+generally speaking native launchers like ``srun`` will not work.
+Most Omni-Path clusters use a workload manager or queueing system
+such a Slurm. To select an appropriate Chapel launcher you can set
+``CHPL_LAUNCHER`` to one of the following values:
+
+===================  ======================================
+Launcher Name        Description
+===================  ======================================
+gasnetrun_ofi         run jobs interactively on your system
+slurm-gasnetrun_ofi   queue jobs using Slurm (srun/sbatch)
+===================  ======================================
+
+.. _setting-ofi-spawner:
+
+-------------------
+Selecting a Spawner
+-------------------
+
+Under the covers ``gasnetrun_ofi`` based launchers must figure out
```

Comment:
"gasnetrun_ofi based" should have a hyphen like "gasnetrun_ofi-based", but I'm not sure if that would look weird since it is partially code?

Reply:
It's a little weird, but not terrible. I changed.